data_pipelines_cli/dbt_cloud_api_client.py

```diff
@@ -0,0 +1,224 @@
+
+    def create_repository(self, project_id, git_clone_url):
+        """
+        Note: After creating  a dbt Cloud repository's SSH key, you will need to add the generated key text as
+         a deploy key to the target repository. This gives dbt Cloud permissions to read / write in the repository.
+
+        :param git_clone_url: Repository remote url
+        :param project_id: ID of the project
+        :return: repository ID and deploy key
+        """
+        new_repository = {
+            "account_id": self.account_id,
+            "project_id": project_id,
+            "remote_url": git_clone_url,
+            "git_clone_strategy": "deploy_key",
+            "github_installation_id": None,
+            "token_str": None
+        }
+
+        new_repository_data = json.dumps(new_repository)
+
+        response = self.request(
+            f"{self.host_url}/v3/accounts/{str(self.account_id)}/projects/{str(project_id)}/repositories/",
+            new_repository_data)
+        return response["data"]["id"], response["data"]["deploy_key"]["public_key"]
+
+    def create_development_environment(self, project_id):
+        """
+        Create development environment
+
+        :param project_id: ID of the project
+        :return: ID of created environment
+        """
+        new_env = {
+            "id": None,
+            "type": "development",
+            "name": "Development",
+            "account_id": self.account_id,
+            "project_id": project_id,
+            "state": 1,
+            "use_custom_branch": False,
+            "custom_branch": None,
+            "dbt_version": "1.0.0",
+            "supports_docs": False,
+        }
+
+        new_env_data = json.dumps(new_env)
+
+        response = self.request(
+            f"{self.host_url}/v3/accounts/{str(self.account_id)}/projects/{str(project_id)}/environments/",
+            new_env_data)
+        return response["data"]["id"]
+
+    def associate_connection_repository(self, name, project_id, connection_id=None, repository_id=None):
+        """
+        Link connection and repository to project
+
+        :param name: Name of the project
+        :param project_id: ID of the project
+        :param connection_id: ID of the connection to be associated
+        :param repository_id: ID of the repository to be associated
+        :return: ID of the project
+        """
+        new_connection = {
+            "name": name,
+            "account_id": self.account_id,
+            "id": project_id,
+            "connection_id": connection_id,
+            "repository_id": repository_id
+        }
+
+        new_connection_data = json.dumps(new_connection)
+        response = self.request(f"{self.host_url}/v3/accounts/{str(self.account_id)}/projects/{str(project_id)}",
+                                new_connection_data)
+
+        return response["data"]["id"]
+
+    def create_credentials(self, schema, project_id):
+        """
+        Create credentials
```

Comment:
This kind of docs makes no sense, name of the method is the same.

Reply:
I've changed description.